src/agents/graph.py

```diff
@@ -81,21 +81,30 @@ class KymaGraph:
 
     plan_parser = PydanticOutputParser(pydantic_object=Plan)  # type: ignore
 
-    def __init__(self, model: IModel, memory: BaseCheckpointSaver):
-        self.model = model
+    planner_prompt: ChatPromptTemplate
+
+    def __init__(self, models: dict[str, IModel], memory: BaseCheckpointSaver):
+        self.models = models
         self.memory = memory
 
-        self.kyma_agent = KymaAgent(model)
-        self.k8s_agent = KubernetesAgent(model)
+        gpt_4o_mini = models[LLM.GPT4O_MINI]
+        gpt_4o = models[LLM.GPT4O]
+
+        # TODO: switch to gpt_4o when necessary
```

Comment:
Is this really a todo, a task we will resolve in the future?

Reply:
we need to change it to gpt_4o as we implement Kyma agent. I think, gpt_4o is not enough for such complex task

Reply:
Okay, that makes sense.